src/components/select/__snapshots__/template.test.js.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Select with dependant components renders with error message 1`] = `""`;
```

Comment:
Is this snapshot correct?

Reply:
Hmm.. doesn't look right. I'll have a look.

Reply:
It seems like `select` is the only component that requires a `label` param in order to render the message - that's why we have an empty string there.
```
{% if params.label %}
{{ govukLabel({...}) }}
{% endif %}
```
I will remove the conditional statement for now, for consistency, and we can discuss the approach and apply it consistently.

Reply:
Updated. I'll leave the commits separated to ease the review. Will rebase afterwards to leave the history clean.